api/src/main/java/com/codeforcommunity/dto/site/ParentRecordStewardshipRequest.java

```diff
@@ -0,0 +1,47 @@
+package com.codeforcommunity.dto.site;
+
+import com.codeforcommunity.exceptions.HandledException;
+
+import java.sql.Date;
+import java.util.List;
+
+public class ParentRecordStewardshipRequest extends RecordStewardshipRequest {
+
+  private Integer childUserId;
+
+  public ParentRecordStewardshipRequest(
+      Date date,
+      boolean watered,
```

Comment:
discussion: Maybe in the future we should put these activities (`watered`, `mulched`, `cleaned`, `weeded`) in an object. Pinging @chromium-52 . Might take significant refactoring though.

Reply:
+1
iirc we do this to mirror how we eventually store this in the DB. Remember our persistence model need not be our program model.

Reply:
That does make sense.